docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,32 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-vm]]
+== Choosing the VM image
+
+When you start Minishift, it downloads an launches a VM image that runs the OpenShift cluster.
+The VM image is a Live ISO that contains an operating system as well as all the dependencies to
+run the OpenShift cluster.
```

Comment:
@thatdocslady I disagree, I strongly feel that we do not need to explain this again here. When an user lands here he is just looking way to start Minishift with CentOS. If he wants to know more about Minishift then he should look in getting started section.